app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    split_com = command.split()
+
+    cp = split_com[0]
+    source_file = split_com[1]
+    destination_file = split_com[2]
```

Comment:
You can use unpacking here
```suggestion
    cp, source_file, destination_file = split_com
```

Reply:
ok, accepted